rasa/core/brokers/broker.py

```diff
@@ -32,6 +32,21 @@ def publish(self, event: Dict[Text, Any]) -> None:
 
         raise NotImplementedError("Event broker must implement the `publish` method.")
 
+    # noinspection PyMethodMayBeStatic
+    def is_ready(self, *args: Any, **kwargs: Any) -> bool:
```

Comment:
mhm, why not:
```suggestion
    def is_ready(self) -> bool:
```

Reply:
thought other subclass of `EventBroker` might use other information to determine ready state - but we can update that later if necessary